### DISCUSSION TITLE
snapcast direct streaming to wifi (dlna/upnp ready) speakers

### DISCUSSION THREAD

**Discussion by sergei-kobets** [snapcast/snapcast#787] [Ideas]
Hi everyone. I will buy two huawei sound x, which literraly have best price/quality/performance ratio. The main problem that both speakers doesnt have opportunity to stream sync audio, because Huawei didnt care about this default feature Thanks god, they are  dlna/upnp ready... I found information that some media players are able to stream music to one Speaker (local dlna server => speaker is renderer). Two or more Speakers will have Problems due to lack of sync..... My main question is: Can snapcast server stream sync audio direct to dlna speakers or not ?

**Comment by darkzbaron** [comment]
Any update on that?

**Comment by dgalli1** [comment]
You can always create a pulseaudio-dlna sink and then just play the music with this, synching playback will probably be next to impossible tho.
